internal/pkg/docker/orchestrator/orchestrator.go

```diff
@@ -43,15 +48,22 @@ type DockerEngine interface {
 	Run(context.Context, *dockerengine.RunOptions) error
 	IsContainerRunning(context.Context, string) (bool, error)
 	Stop(context.Context, string) error
+	Build(ctx context.Context, args *dockerengine.BuildArguments, w io.Writer) error
 }
 
 const (
 	orchestratorStoppedTaskID = -1
 	pauseCtrTaskID            = 0
 )
 
-const (
-	pauseContainerURI = "public.ecr.aws/amazonlinux/amazonlinux:2023"
+const pauseContainerURI = "aws-copilot-pause"
+
+var (
+	// '+' is not a valid character in an image tag
+	pauseContainerTag = strings.Replace(version.Version, "+", "PLUS", -1)
+
+	//go:embed Pause-Dockerfile
+	pauseDockerfile string
```

Comment:
I assume we can use `os.ReadFile(filePath)` to read the content of pause container Dockerfile. Is there any reason for using the embed directive.

Reply:
yeah! `os.ReadFile` reads a fail _on the users machine._ `//go:embed` embeds the file _in to the binary itself._ We don't expect users to have the `Pause-Dockerfile` available on their machine, so we are including it in the binary.